synapse_invite_checker/invite_checker.py

```diff
@@ -167,7 +170,7 @@ def __init__(self, config: InviteCheckerConfig, api: ModuleApi):
         ) as db_conn:
             self.store = InviteCheckerStore(api._store.db_pool, db_conn, api._hs)
 
-        InfoResource(self.config, self.__version__).register(self.resource)
+        InfoResource(self.config, self._TMCI_version).register(self.resource)
```

Comment:
Afaik the "contact" field is removed now, so we could think about doing the same.

Reply:
Ignore this, I was looking at the wrong file